Assets/Mirage/Runtime/NetworkIdentity.cs

```diff
@@ -1101,7 +1101,7 @@ bool AddNewObservers(bool initialize, bool changed)
                     // new observer
                     player.AddToVisList(this);
                     // spawn identity for this conn
-                    ServerObjectManager.ShowForConnection(this, player);
+                    ServerObjectManager.ShowForPlayer(this, player);
```

Comment:
Maybe just have it ShowPlayer. Not sure if we need word for

Reply:
Its showing the NI to the player. `ShowPlayer` implies the player is being shown instead.

Maybe `ShowToPlayer` ?

Reply:
Ya showtoplayer then sounds better